version.sbt

```diff
@@ -1 +1,4 @@
+// update the versions in
+// samples/compile-timeDI/build.sbt
+// samples/runtimeDI/build.sbt
```

Comment:
OMG! 😱

Reply:
Master uses dynver now. And propagates the version.